src/components/adaptive-cards/Component/Component.jsx

```diff
@@ -106,18 +106,27 @@ export default function Component({data, className, style: styleProp}) {
   const classes = [];
   const getClass = (propType, value) => (value ? `wxc-ac-${propType}--${String(value).toLowerCase()}` : '');
   const style = {};
+  const cc = [];
+  const acData = data;
 
   const dataWithDefaults = {...C.acDefaultProps, ...data};
 
   for (const [prop, value] of Object.entries(dataWithDefaults)) {
     const propType = (C.acPropTypes && C.acPropTypes[prop]) || undefined;
+    let child;
 
     switch (propType) {
       case undefined:
         console.log('[Component]', 'Unknown property', prop);
         break;
       case acPropTypes.action:
       case acPropTypes.children:
+        for (child of value) {
+          cc.push({type: 'Spacer', separator: child.separator, spacing: child.spacing || 'default'});
```

Comment:
you can use flatMap https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap